modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/SearchBar.js

```diff
@@ -3,8 +3,8 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-export default function ({namespace: portletNamespace}) {
-	const form = document.getElementById(`${portletNamespace}fm`);
+export default function ({formId}) {
+	const form = document.getElementById(`${formId}`);
```

Comment:
@oliv-yu Just saw this: This can be simplified to `document.getElementById(formId);`

Reply:
Oops, thanks for catching that!! Updated... I squashed it with the original to make it easier to cherry-pick for the hotfix 😅